src/modules/graphics/graphics.c

```diff
@@ -2925,7 +2925,7 @@ void lovrModelGetNodeTransform(Model* model, uint32_t node, float position[4], f
   }
 }
 
-void lovrModelSetNodeTransform(Model* model, uint32_t node, float position[4], float scale[4], float rotation[4], float alpha) {
+void lovrModelSetNodeTransform(Model* model, uint32_t node, float position[3], float scale[3], float rotation[4], float alpha) {
```

Comment:
4 floats is the intent here because lovr currently uses 4 floats for vec3 (helps compiler with vectorization).

Reply:
hm, I figured there was a reason. Like that it complains when passing a 3-component xr vector, but I guess I can cast through `float*` to get it to shut up?

Reply:
oh nevermind, not doing direct copies anymore anyway so this change was completely unnecessasy.